File: Java/LamdaDemo1/src/Demo1.java
public class Demo1 {

	public static void main(String[] args) {
		Service s = (int a, int b) -> {
			return a + b;
		};
		int c = s.add(1, 2);
		System.out.println(c);

	}
}

interface Service {
	public int add(int a, int b);
}

File: cpp/Demo1/Demo1/Demo1.cpp
// Demo1.cpp : Defines the entry point for the console application.
//

#include "stdafx.h"
#include <thread>
#include <iostream>
#include "windows.h"

void my_thread()
{
	while (true){
		puts("hello, world");
		Sleep(1000);
	}
}

int _tmain(int argc, _TCHAR* argv[])
{
	std::thread t(my_thread);
	t.join();
	std::cin.get();
	return 0;
}

